### ISSUE TITLE
Submit PR on react-jsonschema-form repo for accessibility issues

### ISSUE THREAD

**Issue by janette** [GetDKAN/dkan2#324]
https://github.com/rjsf-team/react-jsonschema-form

> form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.

**Comment by janette** [comment]
Upgrading to the v2.0.0-alpha.2 release fixes the text input fields, now only remaining issues are with the 'add item' buttons

![btn](https://user-images.githubusercontent.com/314172/75192815-a22f5a80-571a-11ea-8928-a4b077367e82.png)

**Comment by janette** [comment]
One other issue is that when editing an existing dataset, the values are there for a second then disappear. Seems we need to set the Value differently...?

![form](https://user-images.githubusercontent.com/314172/75194822-744c1500-571e-11ea-903e-c07b444df2c6.png)

**Comment by dharizza** [comment]
This issue has multiple ways of fixing, so I'm listing here the PRs

- PR against rjsf is here https://github.com/rjsf-team/react-jsonschema-form/pull/1714
- PR for updating dkan2 schema is here https://github.com/GetDKAN/dkan2/pull/394
- PR for fixing the `drush dkan-metadata-form:sync` command is here https://github.com/GetDKAN/dkan2/pull/392

**Comment by janette** [comment]
Merged the PRs on dkan2, just waiting for the PR on **rjsf** repo to be merged
